nservicebus/azure-service-bus/configuration/azureservicebusqueueconfig.md

```diff
@@ -35,6 +35,6 @@ NOTE: Most of these values are applied when a queue or topic is created and cann
  * `EnablePartitioning`: Increase overall throughput by allowing to use multiple brokers/stores to handle queues and topics to overcome limitations of a single broker/store at increased cost. Partitioning does reduce number of queues or topics per namespace. Defaults to false.
  * `SupportOrdering`: Specifies whether queues and topics should enable support for message ordering. Defaults to true.
 
-NOTE: `QueueName` and `QueuePerInstance` are obsoleted. Instead, use bus configuration object to specify endpoint name and scale-out option.
+NOTE: AzureServiceBusQueueConfig is only available in Version 6 and below. `QueueName` and `QueuePerInstance` are obsoleted since version 6. Instead, use bus configuration object to specify endpoint name and scale-out option.
```

Comment:
@mikeminutillo not that Version 6 refers to the transport and not the core which is V5 for our V6, afaik the config object has always been named BusConfiguration in Core V5 and below?